trade-finance-manager-ui/templates/accessibility-statement.njk

```diff
@@ -37,12 +37,12 @@
   <h2 class="govuk-heading-m" data-cy="feedback-and-contact-information-heading">Feedback and contact information</h2>
   <p class="govuk-body">If you need information on this website in a different format like accessible PDF, large print, easy read, contact us:</p>
   <ul class="govuk-list accessibility-statement-list">
-    <li>DigitalService.TradeFinance@ukexportfinance.gov.uk </li>
+    <li>{{ CONTACT_US_EMAIL_ADDRESS }}</li>
   </ul>
   <p class="govuk-body">We'll consider your request and get back to you in 2 days.</p>
 
   <h2 class="govuk-heading-m" data-cy="reporting-accessibility-problems-heading">Reporting accessibility problems with this website</h2>
-  <p class="govuk-body" data-cy="contact-dtfs">We're always looking to improve the accessibility of this website. If you find any additional problems not listed above or think we're not meeting accessibility requirements, contact: <a href="mailTo:DigitalService.TradeFinance@ukexportfinance.gov.uk">DigitalService.TradeFinance@ukexportfinance.gov.uk</a>.</p>
+    <p class="govuk-body" data-cy="contact-dtfs">We're always looking to improve the accessibility of this website. If you find any additional problems not listed above or think we're not meeting accessibility requirements, contact: <a href="mailTo:{{ CONTACT_US_EMAIL_ADDRESS }}">{{ CONTACT_US_EMAIL_ADDRESS }}</a>.</p>
```

Comment:
Just to confirm this a tag does not require any formatting class?

Reply:
I wouldnt say so, it follows existing formatting for this page in terms of links. Good spot though.